test_rclcpp/test/test_multithreaded.cpp

```diff
@@ -204,13 +204,13 @@ TEST(CLASSNAME(test_multithreaded, RMW_IMPLEMENTATION), multi_consumer_clients)
       results.push_back(pair.first->async_send_request(pair.second));
     }
     // Wait on each future
-    for (uint32_t i = 0; i < results.size(); i++) {
+    for (uint32_t i = 0; i < results.size(); ++i) {
```

Comment:
Doesn't this line still generate a compiler warning?

Same for all the other loops declaring `i` as `uint32_t`.